frontend/src/components/PaymentConfirmationModal.tsx

```diff
@@ -45,6 +45,7 @@ function PaymentConfirmationModal({
   }, []);
 
   const onConfirm = async () => {
+    // If on demo don't call the backend
```

Comment:
I think this comment is redundant, the code is clear without it

Reply:
Removed